backend/common/src/main/java/ai/verta/modeldb/common/subtypes/KeyValueHandler.java

```diff
@@ -125,11 +129,11 @@ public KeyValueHandler(Executor executor, FutureJdbi jdbi, String fieldType, Str
               Set<String> keySet = new HashSet<>();
               for (final var kv : kvs) {
                 if (kv.getKey().isEmpty()) {
-                  throw new InvalidArgumentException("Empty key");
+                  throw new ModelDBException("Empty key", Code.INVALID_ARGUMENT);
                 }
                 if (keySet.contains(kv.getKey())) {
-                  throw new InvalidArgumentException(
-                      "Multiple key " + kv.getKey() + " found in request");
+                  throw new ModelDBException(
+                      "Multiple key " + kv.getKey() + " found in request", Code.INVALID_ARGUMENT);
```

Comment:
We should probably move the helper exceptions too.

Reply:
PR: https://github.com/VertaAI/modeldb/pull/2758